widget/src/api/user-session/login.ts

```diff
@@ -85,7 +91,8 @@ export class LoginAPI implements LoginAPICodeChallengeStorageAccessor {
     };
   }
 
-  createLoginURL(payload?: string): string {
+  createLoginWebsiteUrl( args: {payload: {path?: string}, targetAuthPage: string}): URL {
```

Comment:
Can we use `targetAuthPage` as the type reference here?
and have you test the `login`? i think we only need to leave it empty to make it pointing to login page,
and beside it, we also have `forgot-password` page in the login website